.env.example

```diff
@@ -159,3 +159,9 @@ TIMEZONE="Pacific Time (US & Canada)"
 
 # Number of failed jobs to keep in the database
 FAILED_JOBS_TO_KEEP=100
+
+# Maximum runtime of background jobs in minutes
+# DELAYED_JOB_MAX_RUNTIME=20
+
+# Amount of seconds for delayed_job to sleep before checking for new jobs
+# DELAYED_JOB_SLEEP_DELAY=10
```

Comment:
Not really, do you think we should have every option 'enabled' with the default value? `SCHEDULER_FREQUENCY` is also commented out, but those three are the only ones